features/support/env.rb

```diff
@@ -52,6 +52,8 @@ def debrand_string(str)
         when "Step 1: Create Kickstart Profile" then str
         when "Create Kickstart Profile" then str
         when "Test Erratum" then str
+        when "errata-cache-default" then str
```

Comment:
@meaksh  we want to get rid of debrand string, look at https://github.com/SUSE/spacewalk-testsuite-base/issues/259